Makefile

```diff
@@ -183,6 +183,10 @@ run-dev-node:
 run-dev-node-release:
 	@ ./scripts/gear.sh run node --release -- --dev
 
+.PHONY: run-dev-node-release-log
+run-dev-node-release-log:
+	@ RUST_LOG="gwasm=debug,pallet_gear=debug,gear_common=debug,gear_core_processor=debug,gear_backend_sandbox=debug" ./scripts/gear.sh run node --release -- --dev -l0
+
```

Comment:
I don't see the reason to add this make binding separately.
You always able to run `RUST_LOG=... make run-dev-node-release`
If main purpose of this bind is to avoid manual typing - provide this universally for all node runs. `gear_common` and `gear_backend_*` are excess.
All pallets and gwasm are only required

Reply:
Yes, it is just a shortcut for a logged run. Fixed the crates list.